project/Dependencies.scala

```diff
@@ -17,7 +17,6 @@ object Dependencies {
   val playCache = PlayImport.cache
   val awsSimpleEmail = "com.amazonaws" % "aws-java-sdk-ses" % awsClientVersion
   val sqs = "com.amazonaws" % "aws-java-sdk-sqs" % awsClientVersion
-  val snowPlow = "com.snowplowanalytics" % "snowplow-java-tracker" % "0.5.2-SNAPSHOT"
```

Comment:
Now that this dependency is being removed you can also remove the 'snapshots' resolver! :confetti_ball: 

It was originally added here... https://github.com/guardian/membership-frontend/pull/191/files#r152640638

Reply:
Done!